### ISSUE TITLE
Update ADODB

### ISSUE THREAD

**Issue by aethelwulffe** [LibreHealthIO/LibreEHR#990]
Required for php 7.1 compatibility (I believe).  Other fixes are relevant to the same.

**Comment by Perseus** [comment]
I've been using the site on php 7.1 for a bit and the only issue I ran into was the AdoDB one while installing. Are there any other known issues?

**Comment by teryhill** [comment]
I think some of the other issues are warning that will have to be addressed as we find them. 

Did you upgrade to the latest AdoDB or just fix the error?

**Comment by Perseus** [comment]
I just fixed the error manually by copying the current adodb xml schema code from their git repo. But updating the whole thing is obviously better. Should I create a PR with the updated AdoDB library?

**Comment by teryhill** [comment]
Yes create the pr . update it on your system also so you can test it .
